google_compute_engine/accounts/oslogin_utils.py

```diff
@@ -105,28 +111,29 @@ def UpdateOsLogin(self, enable, two_factor=False):
       duration: int, number of seconds before updating the NSS cache.
 
     Returns:
-      int, the return code from updating OS Login, or None if not present.
+      int, the return code from updating OS Login, or None if no update.
     """
-    status = self._GetStatus()
+    # Two factor can only be enabled when OS Login is enabled.
+    two_factor = two_factor and enable
+    status = self._GetStatus(two_factor=two_factor)
     if status is None:
       return None
 
-    current_time = time.time()
-    if status == enable:
-      if status and current_time - self.update_time > NSS_CACHE_DURATION_SEC:
+    if enable:
```

Comment:
can we change the variable names to something more descriptive?

Reply:
Sure - made it `enable_oslogin`.